src/utils.py

```diff
@@ -49,9 +49,7 @@ def errcho(x):
     return x
 
 def get_input(message):
```

Comment:
trace what else is using this, no surprises please.

Reply:
a few things were relying on receiving an empty string for 'no input', which is still true, but I'm not 100% sure what would happen when stdin is not available? Indefinite pause? I've switched those to `utils.uin` with an explicit default parameter of `None` so it's no so ambiguous.